applications/GeoMechanicsApplication/custom_conditions/U_Pw_condition.cpp

```diff
@@ -59,19 +59,6 @@ void UPwCondition<TDim, TNumNodes>::CalculateLocalSystem(MatrixType&        rLef
     KRATOS_CATCH("")
 }
 
-//----------------------------------------------------------------------------------------
-
-template <unsigned int TDim, unsigned int TNumNodes>
-void UPwCondition<TDim, TNumNodes>::CalculateLeftHandSide(MatrixType&        rLeftHandSideMatrix,
-                                                          const ProcessInfo& rCurrentProcessInfo)
-{
-    KRATOS_TRY;
-
-    KRATOS_ERROR << "UPwCondition::CalculateLeftHandSide is not implemented" << std::endl;
-
-    KRATOS_CATCH("");
-}
```

Comment:
Just to validate my understanding on why this is done: your new solver calls the CalculateLeftHandSide directly and since the solver (probably?) always works on the entire domain, we cannot have any throwing LHS functions (crashing the run), did I understand this correctly?

Reply:
indeed, plus its perfectly reasonable for a condition to not have values on the lhs, thus returning 0